### DISCUSSION TITLE
Feature: physical properties

### DISCUSSION THREAD

**Discussion by kavonszadkowski** [dfki-ric/phobos#295] [Ideas]
Different physics engines have different ways of dealing with physical properties such as softness of materials. Without any knowledge of the engine, it is impossible to find values in a range useful for certain tasks.
It could be valuable if Phobos possessed functions to assist the user with the choice of at least a range of values for certain applications.

**Comment by Amudtogal** [comment]
This is a great idea. However, for release 1.0. We keep the trouble down to a minimum.
With the new annotation objects, it is basically possible to add annotation information for different physics engines (see the SDF export for an example).

This would be a great feature for any release beyond 1.0.
